examples/vello_editor/src/text.rs

```diff
@@ -300,10 +304,10 @@ impl Editor {
             scene.fill(Fill::NonZero, transform, Color::STEEL_BLUE, None, &rect);
         }
         if let Some(cursor) = self.editor.selection_strong_geometry(1.5) {
-            scene.fill(Fill::NonZero, transform, Color::WHITE, None, &cursor);
+            scene.fill(Fill::NonZero, transform, if self.cursor_blink {Color::TRANSPARENT} else {Color::WHITE}, None, &cursor);
```

Comment:
Since we're painting the whole scene regardless, might as well just add the condition around both of these `if let Some`s.

Reply:
Done.